docs/BuildOptions.md

```diff
@@ -174,275 +153,236 @@ as the `inet_pton` function gets a poor-man's simple replacement that can
 
-This is available only for GNU-compatible compilers.
+When this happens, the caller peer will see a mismatch between the source 
+10.0.1.15 address and the address it tried to contact (i.e. 10.0.1.20). It will 
+be treated as an attack attempt and rejected.
+
+This problem can be slightly mitigated by binding the listening socket to a 
+specific address. So, if you bind it to 10.0.1.20 in the above example, then 
+wherever you try to send a packet over that socket it will always have the 
+source address 10.0.1.20 (and the fix provided by this option will also not 
+apply in this case). However, this problem still exists if the listener socket 
+is bound to the "whole machine" (i.e. set to "any" address).
+
+When the `--enable-pktinfo` option is ON, a mechanism is added to forcefully set 
+the source IP address in a response packet (to 10.0.1.20 in the above example).
+This address is first extracted from the incoming packet as the target address. 
+This fixes the problem, as it will be interpreted by the caller peer correctly.
 
+This feature is turned off by default because the impact on performance is 
+currently unknown. The problem is that it causes the CMSG information to be read 
+from (and set on) every packet when a socket is bound to "any" address. The 
+CMSG information will effectively be extracted from every incoming packet, as 
+long as the socket, through which it is sent or received, is not bound to a
+specific address, including data packets coming in within the frames of an
+existing connection.
 
-`--enable-relative-libpath` (default: OFF)
-------------------------------------------
 
-Enables adding the relative path to a library that allows applications
-to get linked against a shared SRT library by reaching out to a sibling
-`../lib` directory, provided that the library and applications are installed
-in the POSIX/GNU style directories. This might be useful when instaling
-SRT and the applications in some its own directory, where the global library
-path does not lead. There have been reported problems that it may cause
-additionally, so this is OFF by default.
+**`--enable-profile`** (default: OFF)
+
+Enables code instrumentation for profiling.
+
+This is available only for GNU-compatible compilers.
 
 
-`--enable-shared` and `--enable-static` (both default: ON)
-----------------------------------------------------------
+**`--enable-relative-libpath`** (default: OFF)
 
-Enables building SRT as a shared and static library.
+Enables adding a relative path to a library. This allows applications to be
+linked against a shared SRT library by reaching out to a sibling `../lib`
+directory, provided that the library and applications are installed in POSIX/GNU
+style directories. This might be useful when installing SRT and applications
+in a directory, in which the library subdirectory is not explicitly defined
+among the global library paths (such as `/opt/srt-1.4/lib`, for example).
```

Comment:
among the global library paths. Consider, for example, this application and its 
required library:

* `/opt/srt/bin/srt-live-transmit`
* `/opt/srt/lib64/libsrt.so`

By using the `--enable-relative-libpath` option, the `srt-live-transmit` 
application has a relative library path defined inside as `../lib64`. A dynamic 
linker will find the required `libsrt.so` file by this path: `../lib64/libsrt.so`. 
This way the dynamic linkage will work even if `/opt/srt/lib64` path isn't added 
to the system paths in `/etc/ld.so.conf` or in the `LD_LIBRARY_PATH` environment 
variable.